app/policies/permission_policy.rb

```diff
@@ -0,0 +1,2 @@
+class PermissionPolicy < RolePolicy
+end
```

Comment:
Here!!! strange... Role is not Policy. Better EntityPolicy<BasePolicy DictionaryEntityPolicy <DictionaryPolicy ect

Reply:
Use ApplicationPolicy as ancestor